test/python/qobj/test_qobj.py

```diff
@@ -148,7 +149,26 @@ def setUp(self):
                                    ],
                                    qubit_lo_freq=[4.9],
                                    meas_lo_freq=[6.9],
-                                   rep_time=1000),
+                                   rep_time=1000,
```

Comment:
By initializing the valid_qobj in the `setUp` method you are not actually running the test in the test. This should be a dictionary to start, or moved to the actual test.